setup.py

```diff
@@ -32,6 +32,7 @@ def package_data(pkg, roots):
     ],
     install_requires=[
         'XBlock<0.5',
+        'django-crum'
```

Comment:
I'm assuming that all versions of the xblock will have this installed, so I don't pin it. I suppose I should just leave it out if that's my assumption ... though this makes it available for the xblock-sdk.